spec/passaporte_web/service_account_spec.rb

```diff
@@ -130,32 +130,32 @@
     let(:service_account) { PassaporteWeb::ServiceAccount.find("859d3542-84d6-4909-b1bd-4f43c1312065") }
     context "on success" do
       it "should update the ServiceAccount attributes on the server" do
-        service_account.plan_slug.should == 'free'
-        service_account.expiration.should == '2014-04-01 00:00:00'
+        expect(service_account.plan_slug).to eq('free')
+        expect(service_account.expiration).to eq('2014-04-01 00:00:00')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.